pkg/local-gadget-manager/local-gadget-manager_test.go

```diff
@@ -213,6 +236,7 @@ func TestAuditSeccomp(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to start local gadget manager: %s", err)
 	}
+	defer localGadgetManager.Close()
```

Comment:
How does that work? With a defer, it is executed after the call to `checkFdList`.

Reply:
`initialFdList` is gathered *after* `NewManager`, hence this test does not take into consideration fds allocated by the local gadget manager. I make this way because I think this test is about the gadget not leaking fds. `TestClose()` tests the local gadget manager.